components/clubhouse_io/clubhouse.app.js

```diff
@@ -0,0 +1,156 @@
+     * @params {array} storyLinks - An array of story links attached to the story. Each story link must have the [CreateStoryLinkParams](https://clubhouse.io/api/rest/v3/#Body-Parameters-34268) structure.
+     * @params {string} storyType - The type of story (feature, bug, chore).
+     * @params {array} tasks - An array of tasks connected to the story. Each task must have the [CreateTaskParams](https://clubhouse.io/api/rest/v3/#CreateTaskParams) structure.
+     * @params {string} updatedAt - The time/date the story was updated.
+     * @params {integer} workflowStateId - The ID of the workflow state the story will be in.
+     * @returns {story: object } An object with the created story, as per the input provided and default values. See the full schema at [Create Story Responses](https://clubhouse.io/api/rest/v3/#Responses-80269).
+     */
+    async createStory(data) {
+      return await this._withRetries(
+        () => this._clubhouseio().createStory(data),
+      );
+    },
+    /**
+     * Returns a list of all Epics and their attributes.
+     * @returns {epics: array } An array of all epics in the connected Clubhouse account.
+     * See the [Epic schema](https://clubhouse.io/api/rest/v3/#Epic) at the API docs.
+     */
+    async listEpics() {
+      return await this._withRetries(
+        () => this._clubhouseio().listEpics(),
+      );
+    },
+    /**
+     * Returns a list of all Epics and their attributes.
+     * @returns {interations: array } An array of all interations in the connected Clubhouse
+     * account.
+     * See the [Iteration schema](https://clubhouse.io/api/rest/v3/#Iteration) at the API docs.
+     */
+    async listIterations() {
+      return await this._withRetries(
+        () => this._clubhouseio().listIterations(),
+      );
+    },
+    /**
+     * Returns a list of all Members and their attributes.
+     * @returns {members: array } An array of all members in the connected Clubhouse account.
+     * See the [Members schema](https://clubhouse.io/api/rest/v3/#Member-1426) at the API docs.
+     */
+    async listMembers() {
+      return await this._withRetries(
+        () => this._clubhouseio().listMembers(),
+      );
+    },
+    /**
+     * Returns a list of all Workflows and their attributes.
+     * @returns {workflows: array } An array of all Workflows in the connected Clubhouse account.
+     * See the [Workflow schema](https://clubhouse.io/api/rest/v3/#Workflow) at the API docs.
+     */
+    async listWorkflows() {
+      return await this._withRetries(
+        () => this._clubhouseio().listWorkflows(),
+      );
+    },
+    /**
+     * Searches for stories in your clubhouse.
+     * @params {string} query - The search query based on the [Search page](https://help.clubhouse.io/hc/en-us/articles/115005967026) [search operators](https://help.clubhouse.io/hc/en-us/articles/360000046646-Search-Operators) to use for finding stories.
```

Comment:
i did notice some lines exceeded the 100 chars max but noted a loophole in ESLint validation when there was a URL exceeding this limit. I have send the URLs to a new lines, in somecases breaking the markdown [text](url) since this is not being rendered anyway.